Codeinwp/optimole-wp
# Feat/no js logic

### All Submissions:
 

### Changes proposed in this Pull Request:
Add a filter for the HTML tag to allow adding additional classes.
Changed the logic so that `optml_no_js` class is added by default on the HTML tag and is removed upon JS execution.


### How to test the changes in this Pull Request:

1. Activate the plugin.
2. Check that with JS enabled there is no `optml_no_js` class on the HTML tag and everything is working as expected.
3. Check with JS disabled that the class `optml_no_js` is present on the HTML tag and the placeholder images are not being displayed. The `<noscript>` tags should be used instead.

### Other information:
